packages/api/src/command/medical/patient/get-patient-facility-matches.ts

```diff
@@ -51,67 +58,58 @@ async function getCqFacilityMatches(cqLinks: CQLink[]): Promise<PatientFacilityM
   const patientFacilityMatches: PatientFacilityMatch[] = [];
 
   for (const cqLink of cqLinks) {
+    if (!cqLink.patientResource) {
+      continue;
+    }
+
     const cqFacility = await CQDirectoryEntryModel.findOne({
       where: { id: cqLink.oid },
     });
 
-    if (!cqLink.patientResource) {
+    if (!cqFacility) {
       continue;
     }
 
     const patientMatchDemo = cqPatientToPatientMatch(cqLink.patientResource);
 
-    if (cqFacility) {
-      patientFacilityMatches.push({
-        name: cqFacility.name ?? "",
-        oid: cqFacility.id ?? "",
-        address: {
-          addressLine1: cqFacility.addressLine ?? undefined,
-          city: cqFacility.city ?? undefined,
-          state: cqFacility.state as USState,
-          zip: cqFacility.zip ?? undefined,
-        },
-        patient: patientMatchDemo,
-      });
-    }
+    patientFacilityMatches.push({
+      name: cqFacility.name ?? "",
+      oid: cqFacility.id ?? "",
+      address: {
+        addressLine1: cqFacility.addressLine ?? undefined,
+        city: cqFacility.city ?? undefined,
+        state: (cqFacility.state as USState) ?? undefined,
+        zip: cqFacility.zip ?? undefined,
+      },
+      patient: patientMatchDemo,
+    });
   }
 
   return patientFacilityMatches;
 }
 
 function cqPatientToPatientMatch(patient: PatientResource): PatientFacilityMatchDemo {
   const givenNames = patient.name[0].given;
-  const firstName = givenNames?.join(" ") ?? "";
+  const firstName = givenNames.join(" ");
 
   const familyName = patient.name[0].family;
```

Comment:
This can also be `undefined`, and toTitleCase doesn't expect `undefined` and would result in runtime error